openapi/components/schemas/CreditMemos/CreditMemo.yaml

```
@@ -0,0 +1,150 @@
+type: object
+description: Credit memo object.
+required:
+  - customerId
+  - currency
+properties:
+  id:
+    description: Credit memo ID.
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  invoiceId:
+    description: ID of the invoice which the credit memo is issued upon.
+    nullable: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  customerId:
+    description: Customer's ID.
+    allOf:
+      - $ref: "#/components/schemas/ResourceId"
+  number:
+    description: An auto-incrementing number based on the sequence of credit memos for any particular customer.
+    readOnly: true
+    type: integer
+  items:
+    type: array
+    items:
+      type: object
+      required:
+        - unitPrice
+      properties:
+        invoiceItemId:
+          description: Credit memo item's description.
```

```suggestion
          description: An invoice item's ID that the credit item is referenced to.
```
Or something like that

---

I'm not sure it is, we're adding a related `invoiceId`, why not item right away

---

OKay, I see, yes